Hurd/ContentView.swift

```diff
@@ -11,7 +11,7 @@ import FirebaseAuth
 struct ContentView: View {
     
     @State var selection = 0
-    @State var addTripFormPresented: Bool = false
+    @ObservedObject var vm = AddTripFormViewModel()
```

Comment:
This should be a  a StateObject because we are creating the instance in this view. if we were to pass this instance of a class down the view hierarchy then we would use the @ObservedObject Property wrapper. 

---
Use the state object property wrapper to ensure consistent results when the current view creates the observed object. Whenever you inject an observed object as a dependency, you can use the @ObservedObject.
```swift 
@StateObject var vm = AddTripFormViewModel()
```